tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "stripInternal": true,
     "outDir": "dist",
     "allowJs": false,
+    "skipLibCheck": true,
```

Comment:
What's the rationale for this change?

Reply:
It looks like travis was always trying to install the latest typescript 3.7.3 where we are only on 2.7.2 for now.